src/metabase/models/params/custom_values.clj

```diff
@@ -0,0 +1,110 @@
+(ns metabase.models.params.custom-values
```

Comment:
I see it's to differentiate it from `field-values`, which is the default type of parameter values source. Perhaps we should avoid creating a name for something that doesn't really have meaning (by that I mean, there's no logical similarity between source=card and source=list other than the fact that it's not source=field). I'd suggest separating these blocks into separate namespaces.

Reply:
that's a good point. I name it custom-values solely because the project name is custom-values.
Didn't give it much thought.

I'll move this into `metabase.models.params.field-values` 👍

Reply:
fair point. I named it custom-values because it's the project name. admittedly it was not given enough thought.

I created a ns `card-values` and move the code there instead. since `static-list` code is only the `query-matches` function, I'll move it to the api.dashboard for now.

if it grows we will have a dedicated ns for it.

https://github.com/metabase/metabase/pull/27343/commits/373301925d5902d51d0ab1f5e9f2b8f55e496f08